x-pack/legacy/plugins/code/public/monaco/override_monaco_styles.scss

```diff
@@ -1,27 +1,25 @@
-.codeContainer__editor {
-  .monaco-editor .cursors-layer > .cursor {
-    display: none !important;
-  }
+.monaco-editor .cursors-layer > .cursor {
+  display: none !important;
+}
 
-  textarea.inputarea {
-    display: none !important;
-  }
+textarea.inputarea {
+  display: none !important;
```

Comment:
Removing the outer-most selector makes some of these other selectors dangerous like this one. Is there any way to scope all this file to Code specifically?